src/utils/globals.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+export default function setElementsVersion() {
+    if (window.Box) {
+        window.Box.Elements = {
```

Comment:
It may be a bit safer to lazily create the `Elements` object and add defaults to avoid the eslint issues:

```
window.Box.Elements = window.Box.Elements || {};
window.Box.Elements.version = __VERSION__ || '';
```